examples/compiler/00_index.ipynb

```diff
@@ -0,0 +1,53 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The Devito's Operator \n",
+    "\n",
+    "In Devito, an `Operator` carries out three fundamental tasks: generation of low-level code, JIT compilation, and execution. Overall, the lowering process performed by an `Operator` – from high-level equations to dynamically compiled and executable code – consists of multiple compiler passes, summarized in the Figure bellow. \n",
+    "\n",
+    "![Compiler Stages](https://i.ibb.co/dKjsZ72/compiler-stages.png)\n",
```

Comment:
I'd still prefer it to be a pdf, and I'd make it smaller

Reply:
done